.github/workflows/end2end.yaml

```diff
@@ -355,12 +355,22 @@ jobs:
         shell: bash
         run: |-
           echo "CTST_TAG=$(sed 's/.*"cli-testing": ".*#\(.*\)".*/\1/;t;d' ./tests/ctst/package.json)" >> $GITHUB_ENV
+      - name: Extract environment
+        run: |-
+          sudo curl --fail -L https://github.com/mikefarah/yq/releases/download/${{ env.YQ_VERSION}}/${{ env.YQ_BINARY }} -o /usr/bin/yq
+          sudo chmod +x /usr/bin/yq
+          cd solution/
+          cat <<EOF >> $GITHUB_ENV
+          SORBET_TAG=$(yq eval '.sorbet.tag' deps.yaml)
+          EOF
```

Comment:
As this duplicates the kafka step when installing yq, I would suggest creating a dedicated action file for that, that can be shared with `build-kafka`.

Reply:
You can also avoid the `cd` by setting the `working-directory` parameter to the solution folder

Reply:
Adding a specific file just for those 2 lines of installing yq feels a bit overkill to me
I now use `working-directory` though